### PR TITLE
Port To GLib

### PR DESCRIPTION
**Pull request by aryanharitsh123**
Completed and working.

### CODE REVIEW

File: sugargame/event.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import logging
+import os
```
Comment (rhl-bthr) [inline sugargame/event.py]: Not Required.

File: sugargame/event.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
 from gi.repository import Gdk
+from gi.repository import GLib
 from gi.repository import GObject
```
Comment (rhl-bthr) [inline sugargame/event.py]: Needs to be removed

**Review by rhl-bthr — CHANGES_REQUESTED** [review]
Also, please squash your commits into one with an appropriate commit message

**Comment by Hrishi1999** [comment]
Please squash commits

**Pull request closed without merging**